teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './main'
+
+class Teacher < Person
+  def initialize(name, age, specialization)
```

Comment:
okay, but I thought since this new class is a Teacher... maybe they don't need their parent's permission? or do they need it?

Reply:
@Juanse7793, I knew that was your thoughts. The teacher does not actually require permission from the parents because it will be difficult to find a teacher of age less than 18. We are just assuming that it can be possible.